aws_lambda_builders/workflows/ruby_bundler/workflow.py

```diff
@@ -0,0 +1,55 @@
+"""
+Ruby Bundler Workflow
+"""
+
+from aws_lambda_builders.workflow import BaseWorkflow, Capability
+from aws_lambda_builders.actions import CopySourceAction
+from .actions import RubyBundlerInstallAction, RubyBundlerVendorAction
+from .utils import OSUtils
+from .bundler import SubprocessBundler
+
+
+class RubyBundlerWorkflow(BaseWorkflow):
+
+    """
+    A Lambda builder workflow that knows how to build
+    Ruby projects using Bundler.
+    """
+    NAME = "RubyBundlerBuilder"
+
+    CAPABILITY = Capability(language="ruby",
```

Comment:
we can add validation of ruby runtime in another PR.

Reply:
Yep, I commented in that other PR what Ruby would need. It's a really simple one-liner.